thinc/layers/remap_ids.py

```diff
@@ -31,11 +41,22 @@ def forward(
     table = model.attrs["mapping_table"]
     default = model.attrs["default"]
     dtype = model.attrs["dtype"]
+    column = model.attrs["column"]
+    if column is not None:
+        inputs = cast(Ints2d, inputs)
+        inputs = inputs[:, column]
+    # elements of cupy arrays are 0-dimensional arrays
+    # not the integers stored in the original mapper.
+    if is_cupy_array(inputs):  # type: ignore
+        inputs = to_numpy(inputs)  # type: ignore
```

Comment:
```suggestion
    inputs = to_numpy(inputs)  # type: ignore
```

`to_numpy` already checks if the array is a CuPy array, so now we'd be checking it twice in case it is. So, I think it's better to just call `to_numpy_ unconditionally.

I also wonder if we can avoid the `type: ignore` here, or if not make it a more specific ignore.